editor/src/components/titlebar/title-bar.tsx

```diff
@@ -28,8 +26,8 @@ const ProjectTitle: React.FC<React.PropsWithChildren<ProjectTitleProps>> = ({ ch
   return <div style={{ fontWeight: 400, fontSize: 12, padding: '0 10px' }}>{children}</div>
 }
 
-const TitleBar: React.FC<React.PropsWithChildren<TitleBarProps>> = () => {
-  const { loginState, projectName, isCodeEditorVisible, isPreviewPaneVisible, dispatch } =
+const TitleBar = React.memo(() => {
```

Comment:
this seems to be an unrelated fix that sneaked in :)

Reply:
eh yeah I caught it after I pushed 😅